edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/KubernetesEnvironmentOperator.cs

```diff
@@ -12,6 +12,7 @@ namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes
 
     public class KubernetesEnvironmentOperator : IKubernetesEnvironmentOperator
     {
+        const int TimeoutSeconds = 180;
```

Comment:
Why this number? Any chance to use some parameter or config value?

Reply:
3 minutes is consistently short enough to avoid the bug in the C# k8s SDK, but allowing it to be configurable is a really good idea.